code/__DEFINES/atmospherics/atmos_core.dm

```diff
@@ -68,14 +68,16 @@
 /// The total visible states
 #define TOTAL_VISIBLE_STATES (FACTOR_GAS_VISIBLE_MAX * (1 / MOLES_GAS_VISIBLE_STEP))
 
-//REACTIONS
-//return values for reactions (bitflags)
-///The gas mixture is not reacting
+// REACTIONS
+// Return values for reactions (bitflags).
+/// The gas mixture is not reacting.
 #define NO_REACTION 0
```

Comment:
NONE?

Reply:
I forgot about this. I made this PR months ago but only decided to finish the last 1% today. I don't fully remember the justification. Don't remember if anything expects NO_REACTION to be falsy. I also assume 0 is just 00000000000000000000000 for the bitflags. Maybe it would make more sense to remove NO_REACTION.